docs/cookbook/platform-sh-advanced.rst

```diff
@@ -0,0 +1,194 @@
+Advanced topics
+---------------
```

Comment:
I think that the build is failing because this document does not have any title.
Let's do:
```
Advanced Platform.sh configurations
===========================
```
:)